tests/gpio/obc.overlay

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2025 The FINCH CubeSat Project Flight Software Contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/
+{
+	gpio_pins
+	{
```

Comment:
It's convention for the format to be this,

```
/ {
    gpio_pins {
```

Reply:
Thanks for catching this! I guess checkpatch.pl doesn't enforce this, or maybe we have that check turned off by mistake.